serverless/src/functions/apps/handler.ts

```diff
@@ -139,7 +137,10 @@ export const submitAppInfo = async (
           throw error;
         }
       }
-    } while (maxTries > 0);
+
+      errorOccurred = false; // No error occurred if this line is reached
```

Comment:
I think it probably needs to be set to `true` when the name is already takes error is returned from the bunny API. and I need to change line 143 to `} while (maxTries > 0 && errorOccurred)`

from reviewing the code now, I don't think it stops before running out of possible tries so it may create more than one instance of the pullzone, but looking at the test we did the other day, I notice it worked as expected without creating unnecessary pullzones. I'll look into this more and reply. It may or may not need to change.

Reply:
For the case that the name is already taken, the loop is gonna continue till `maxRetries = 0` based on the code, so I suggest to review it a bit

Reply:
Well, that is the purpose of the code. We want to regenerate new names 🤔